0001-interledger-architecture/0001-interledger-architecture.md

```diff
@@ -1,136 +1,168 @@
 ---
 title: Interledger Architecture
-draft: 4
+draft: 5
 ---
 # Interledger Architecture
 
 Interledger provides for secure payments across multiple assets on different ledgers. The architecture consists of a conceptual model for interledger payments, a mechanism for securing payments, and a suite of protocols that implement this design.
 
-The [Interledger Protocol (ILP)](#interledger-layer) is the core of the Interledger protocol suite.
+The [Interledger Protocol (ILP)](#interledger-layer) is the core of the Interledger protocol suite. Colloquially, the whole Interledger stack is sometimes referred to as "ILP". Technically, however, the Interledger Protocol is only one layer in the stack.
 
-Colloquially, the whole Interledger stack is sometimes referred to as "ILP". Technically, however, the Interledger Protocol is only one layer in the stack.
+Interledger is not a blockchain, a token, nor a central service. Interledger is a standard way of bridging financial systems. The Interledger architecture is heavily inspired by the Internet architecture described in [RFC 1122](https://tools.ietf.org/html/rfc1122), [RFC 1123](https://tools.ietf.org/html/rfc1123) and [RFC 1009](https://tools.ietf.org/html/rfc1009).
 
-The Interledger architecture is heavily inspired by the Internet architecture described in [RFC 1122](https://tools.ietf.org/html/rfc1122), [RFC 1123](https://tools.ietf.org/html/rfc1123) and [RFC 1009](https://tools.ietf.org/html/rfc1009).
+## Core Concepts
 
-## Interledger Model
+### The Interledger
 
-![Interledger Model: Sender, Connectors, Receiver](../shared/graphs/interledger-model.svg)
+The "Interledger protocol suite" can be used among any network of participants, public or private. There is no single network that all parties must connect to to use these protocols.
 
-### Connectors
+"The Interledger" is the name of a public instance of the Interledger protocol suite which seeks to provide a coherent global financial infrastructure. Ideally, anyone connected to the Interledger should be able to transact with anyone else, each using the underlying systems and currencies of their choice.
```

Comment:
You'd have to ask @michielbdejong. This paragraph is [cribbed from #412](https://github.com/interledger/rfcs/pull/412/files#diff-ce1d0a4db7a75b9030d0436e521d092bR54). I simply moved it higher up in the document.

Reply:
I think it meant something like on the one hand same protocol is used everywhere, and on the other, everywhere is reachable from anywhere.

Btw I also wrote a "v4" of the Interledger whitepaper after that https://github.com/interledger/rfcs/pull/424. We never merged that because I couldn't get all the co-authors to submit their reviews, but maybe you also want to take inspiration from there, or even continue where I left it / merge IL-RFC-1 with the v4 whitepaper?